src/openai/_utils/_logs.py

```diff
@@ -23,3 +25,15 @@ def setup_logging() -> None:
         _basic_config()
         logger.setLevel(logging.INFO)
         httpx_logger.setLevel(logging.INFO)
+
+
+class APIKeyFilter(logging.Filter):
```

Comment:
nit: thoughts on this naming?
```suggestion
class SensitiveHeadersFilter(logging.Filter):
```

Reply:
updated to SensitiveHeadersFilter